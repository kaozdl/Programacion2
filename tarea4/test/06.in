# conjunto
construir_cnj (4,a) .
excluir b c .
excluir c a m .
ops_cnj (2,m) .
construir_cnj (4,a) (2,b) (3,c) (1,d) (5,e) (1,f) (7,g) (1,h) (4,i) (2,j) (3,k) (1,l) (5,m) (1,n) (7,o) .
excluir i o c .
excluir c a z .
ops_cnj (2,m) (5,z) (6,b) .
Fin
