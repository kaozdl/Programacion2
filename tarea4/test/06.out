1>
2>3>4>Vacio.
5>1
0
6>7>8>9>1
2
10>Fin.
